### PR TITLE
fix dose not delete network policy when job finished

### PR DESCRIPTION
**Pull request by wpeng102**
Fix issue https://github.com/volcano-sh/volcano/issues/1177

### CODE REVIEW

**Comment by wpeng102** [comment]
SVC job plugin will create network policy automatically but miss to delete it when job finished.

File: pkg/controllers/job/plugins/svc/svc.go [hunk 1]
```diff
@@ -168,6 +168,14 @@ func (sp *servicePlugin) OnJobDelete(job *batch.Job) error {
 	}
 	delete(job.Status.ControlledResources, "plugin-"+sp.Name())
 
+	if !sp.disableNetworkPolicy {
+		if err := sp.Clientset.KubeClients.NetworkingV1().NetworkPolicies(job.Namespace).Delete(context.TODO(), job.Name, metav1.DeleteOptions{}); err != nil {
+			if !apierrors.IsNotFound(err) {
+				klog.Errorf("Failed to delete Network policy of Job %v/%v: %v", job.Namespace, job.Name, err)
+				return err
```
Comment (k82cn) [inline pkg/controllers/job/plugins/svc/svc.go:175]: what will happen if failed to delete network policy?

Comment (wpeng102) [reply pkg/controllers/job/plugins/svc/svc.go:175]: network policy will left and controller print error log.  No impact to job finish process.
When job deleted, this network policy also be deleted by GC.

**Pull request merged**